src/plugins/intel_cpu/src/transformations/snippets/x64/pass/eliminate_brgemm_copy_b.cpp

```diff
@@ -15,15 +15,22 @@
 
 namespace ov::intel_cpu {
 
-pass::EliminateBrgemmCopyB::EliminateBrgemmCopyB() {
-    MATCHER_SCOPE(EliminateBrgemmCopyB);
+bool pass::EliminateBrgemmCopyB::run_on_model(const std::shared_ptr<ov::Model>& model) {
+    RUN_ON_MODEL_SCOPE(EliminateBrgemmCopyB);
+    OV_ITT_SCOPED_TASK(ov::pass::itt::domains::SnippetsTransform, "ov::intel_cpu::pass::EliminateBrgemmCopyB")
+
     auto m_param = ov::pass::pattern::wrap_type<ov::op::v0::Parameter>();
     auto m_rank_norm = ov::pass::pattern::optional<ov::snippets::op::RankNormalization>(m_param);
-    auto m_copy_b = ov::pass::pattern::wrap_type<BrgemmCopyB>({m_param});
```

Comment:
Hm, it seems like there is some code style check issue: `m_rank_norm` was unused, but there were no indications
Anyway, thanks for the fix :)